### PR TITLE
WT boards: extract `g_is31fl3736_leds` from wt_mono_backlight

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

To further allow for these to be converted to LED Matrix.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Link doesn't like the lack of copyright headers

File: keyboards/wilba_tech/wt60_a/wt60_a.c [hunk 1]
```diff
@@ -0,0 +1,117 @@
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt60_a/wt60_a.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"
```

File: keyboards/wilba_tech/wt65_a/wt65_a.c [hunk 2]
```diff
@@ -0,0 +1,117 @@
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt65_a/wt65_a.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"
```

File: keyboards/wilba_tech/wt65_b/wt65_b.c [hunk 3]
```diff
@@ -0,0 +1,117 @@
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt65_b/wt65_b.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"
```

File: keyboards/wilba_tech/wt75_a/wt75_a.c [hunk 4]
```diff
@@ -0,0 +1,117 @@
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt75_a/wt75_a.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"
```

File: keyboards/wilba_tech/wt75_b/wt75_b.c [hunk 5]
```diff
@@ -0,0 +1,117 @@
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt75_b/wt75_b.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"
```

File: keyboards/wilba_tech/wt75_c/wt75_c.c [hunk 6]
```diff
@@ -0,0 +1,117 @@
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt75_c/wt75_c.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"
```

File: keyboards/wilba_tech/wt80_a/wt80_a.c [hunk 7]
```diff
@@ -0,0 +1,117 @@
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt80_a/wt80_a.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"
```

**Comment by fauxpark** [comment]
Just the existing keymaps failing lint now.

**Pull request merged**